partner_operating_unit/views/res_partner_category_view.xml

```diff
@@ -0,0 +1,34 @@
+<!--
```

Comment:
Please add search view as well, to filter by operating unit when you are able to see many operating units.

Reply:
There is no search defined on this model *yet* and I was afraid we would add it first. To me, the search would be on the module ```base```, and we should inherit from the original search and add the search over operating units.

An intermediate solution would be to add the original search on ```operating_units``` and then inherit from here and add the search for operating units. What do you think?

Reply:
In base there is no view because the tree view just shows the display_name, then the automatic search view is enough. I think we can create a new search view in this new module  just allowing to search in name and operating unit.